map_it_test.go

```diff
@@ -789,11 +794,61 @@ func TestMap_EntryNotifiedEvent(t *testing.T) {
 	})
 }
 
+func TestMap_EntryNotifiedEventWithAddListener(t *testing.T) {
+	it.MapTester(t, func(t *testing.T, m *hz.Map) {
+		const totalCallCount = int32(100)
+		callCount := int32(0)
+		subscriptionID, err := m.AddListener(context.Background(), hz.MapListener{
+			EntryAdded: func(event *hz.EntryNotified) {
+				if event.EventType == hz.EntryAdded {
```

Comment:
How about calling `t.Fatalf("unexpected event type: %v", event.EventType)` if `EventType!= hz.EntryAdded`? That would get rid of the `else` block. Same comment for other similar code.

Reply:
A matter of styling I guess but ok I will do that